src/main/java/com/sprint/mission/discodeit/controller/openapi/UserApiDocs.java

```diff
@@ -0,0 +1,97 @@
+package com.sprint.mission.discodeit.controller.openapi;
+
+import com.sprint.mission.discodeit.dto.user.CreateUserRequest;
+import com.sprint.mission.discodeit.dto.user.CreateUserResponse;
+import com.sprint.mission.discodeit.dto.user.UserResponseDto;
+import com.sprint.mission.discodeit.dto.user.UserUpdateDto;
+import com.sprint.mission.discodeit.error.ErrorResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+
+
+@Tag(name = "User API", description = "User 작업")
+public interface UserApiDocs {
+
+  @Operation(summary = "User 조회", description = "Find single user using userId")
+  @ApiResponses({
+      @ApiResponse(responseCode = "200", description = "User of userId found"),
+      @ApiResponse(
+          responseCode = "404",
+          description = "No such user of userId found",
+          content = @Content(
+              mediaType = "application/json",
+              schema = @Schema(implementation = ErrorResponse.class)
+          )
+      )
+  })
+  ResponseEntity<UserResponseDto> getUser(
+      @Parameter(description = "User ID", required = true)
+      String id
+  );
+
+  @Operation(summary = "User 등록", description = "새로운 사용자를 등록합니다.")
+  @ApiResponses({
+      @ApiResponse(responseCode = "201", description = "User Created"),
+      @ApiResponse(
+          responseCode = "400",
+          description = "User with same email/username exists",
+          content = @Content(
+              mediaType = "application/json",
+              schema = @Schema(implementation = ErrorResponse.class)
+          )
+      )
+  })
+  public ResponseEntity<CreateUserResponse> createUser(
```

Comment:
인터페이스는 기본적으로 public이기에 접근제어자를 적어주지 않아도 될 것 같아요.  
```suggestion
  ResponseEntity<CreateUserResponse> createUser(
```